### PR TITLE
Return cloudferry box

### PR DESCRIPTION
**Pull request by mgrygoriev**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
$PULL_STATUS
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1557/

**Comment by MirantisWorkloadMobility** [comment]
Merge check failed
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1558/

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1559/

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1560/

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request
http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/1561/

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request

File: devlab/Vagrantfile [hunk 1]
```diff
@@ -63,6 +69,12 @@ Vagrant.configure(2) do |config|
             thisnode.vm.provision "shell", path: "./provision/cleanup_nova_instances.sh"
           end
           thisnode.vm.provision "shell", path: "./provision/libvirt.sh"
+        when "lab"
+          thisnode.vm.provision "shell",
+            path: "./provision/cloudferry.sh"
+          thisnode.vm.provision "file",
+            source: "~/.ssh/id_rsa",
+            destination: "/home/vagrant/.ssh/id_rsa" unless not File.exist?("~/.ssh/id_rsa")
```
Comment (roman-verchikov) [inline devlab/Vagrantfile]: Maybe `if File.exist?()`? Double negation is very difficult to read (unless + not). Also, what happens if `~/.ssh/id_rsa` does not exist?

Comment (mgrygoriev) [inline devlab/Vagrantfile]: changed "unless + not" to "if"

**Comment by roman-verchikov** [comment]
Looks good, +1

**Comment by MirantisWorkloadMobility** [comment]
There were no python files updated. Admins, please, check pull request.

**Comment by roman-verchikov** [comment]
Can be merged

**Pull request merged**